participants/shayan

```diff
@@ -0,0 +1,50 @@
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Shayan",
+		"familyName": "ypn",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": false,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": true
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "yazdanpanahH",
+	// company name (optional)
+	"company": "ACME",
```

Comment:
Please add your real company here or remove it if you're not affiliated with a company right now